data/game_config.cfg

```diff
@@ -33,8 +33,10 @@
     hp_bar_scaling=0.6
     xp_bar_scaling=0.5
 
-    # zoom factors must be a sorted list of ascending multipliers
-    zoom_levels = 0.25, 0.33333333333333333, 0.5, 0.75, 1.0, 1.25, 1.5, 2.0, 3.0, 4.0
+    # zoom_levels must be an ascending list of hex sizes in pixels.
+    # hex sizes must be divisible by 4 to prevent graphical glitches.
+    # sizes are in draw space, so standard 1x zoom is always 72.
```

Comment:
Maybe worth highlighting that `tile_size = 72` is defined in `game_config.cpp` and commenting these sizes correspond to 1/4, 1/3, 1/2, 3/4, 1, 1.25, 1.5, 2, 3, 4 times magnification.

Just so it's clear to someone coming back to this later. It may be obvious to us now but easy to forget things months/years down the track.

Reply:
Good idea, i'll add those in.

Looking at it made me realize i missed a zoom level... will bring that up in the main thread